extensions/db/src/main/java/org/mule/extension/db/internal/domain/connection/DbConnectionProvider.java

```diff
@@ -90,7 +90,7 @@
   private DataSource dataSource;
 
   @Override
-  public final DbConnection connect() throws ConnectionException {
```

Comment:
why did you remove the final modifier? You didn't override this method in any CP